visualization/app/codeCharta/ressources/fixed-folders/example.json

```diff
@@ -0,0 +1,386 @@
+{
+	"projectName": "example-project",
+	"apiVersion": "1.1",
```

Comment:
```suggestion
	"apiVersion": "1.2",
```

Reply:
I'd like to keep the versioning and the validation in the other branch. Otherwise the current validation would throw an error, because the `.cc.json` is too new for the application.